samples/schema-registry-integration/README.adoc

```diff
@@ -0,0 +1,133 @@
+== Spring Cloud Stream and Schema Evolution in Action with Kafka Binder.
+
+These are a set of Spring Boot applications to demonstrate Schema Evolution using Spring Cloud Stream with Kafka binder.
+Producer V1 (`producer1`), Producer V2 (`producer2`), and Consumer (`consumer`) are included in this project.
+
+=== Requirement
+As a developer, I'd like to design my consumer to be resilient to differing payload schemas.
+
+=== Assumptions
+For this demonstration, we will simply assume there are two producers producing events with different payload schemas.
+A consumer that consumes both the payload versions will be designed to adapt to evolving schemas.
+
+These applications integrate with Spring Cloud Stream Scheam Registry.
+Both producers and consumers interact with this Schema Registry to register and evolve the schema.
+
+[[build-apps]]
+=== Building the applications
+To build the applications simply execute the following command from the `schema-registry-integration` directory:
+[source,bash]
+----
+./mvnw clean install
+----
+NOTE: The apps can be built and run from w/in an IDE (such as IntelliJ) but you will need to invoke the Maven `package` goal and then `refresh` the project as the Avro Maven plugin needs to execute so that it generates the required model classes - otherwise you will see compile failures around missing `Sensor` class.
+
+[[run-apps]]
+=== Running the applications
+
+==== Pre-requisites
+****
+* The components have all been built by following the <<build-apps>> steps.
+* Apache Kafka broker available at `localhost:9092`
+
+TIP: The included link:../../../tools/kafka/docker-compose/README.adoc#_all_the_things[Kafka tools] can be used to easily start a broker locally on the required coordinates
+****
+
+If you want to use schema registry with Postgres or MYSQL, either start them locally or use the provided docker-compose scripts above.
```

Comment:
> or use the provided docker-compose scripts above.

So that README does not talk about `postgres` or `mysql`. We should add a section in there for these 2 new additions. Also, this would be influenced by the outcome [here](https://github.com/spring-cloud/spring-cloud-stream/pull/2498/files#r957741938).


I would reword this slightly such as:
```
By default, the schema registry is backed by an H2 database. If you want to instead use Postgres, it must be available at `localhost:5432`. Similarly, if you want to use MySQL it must be available at `localhost:3306`. 

TIP: The included link:../../../tools/kafka/docker-compose/README.adoc#_my_new_section_goes_here[Kafka tools] can be used to easily start a broker locally on the required coordinates.

```